docs-markdown/__mocks__/vscode.js

```diff
@@ -0,0 +1,73 @@
+const languages = {
```

Comment:
Hey @adunndevster , do we still need the mocks directory if we're moving away from Jest?

Reply:
Good point... we don't.

Reply:
Fixed!